Assets/Scripts/Networking/GameManager.cs

```diff
@@ -35,18 +35,28 @@ public class GameManager : MonoBehaviourPunCallbacks
 
 		private GameObject instance;
 
-        [Tooltip("The prefab to use for representing the player")]
+
```

Comment:
what's up with the tabbing here? xd

Reply:
It looks fine in VS, so it's just GitHub doing its things